README.md

```diff
@@ -1,5 +1,213 @@
 
-Bowling Challenge
+RACHEL GRIFFITHS
+=================
+
+## CHALLENGE: BOWLING SCORECARD
+------------
+
+##### BUSINESS LOGIC
+* Create a Bowling Scorecard
+* User enters score between 0 and 10 for each roll
+* Each pin knocked down accounts for one point
+* The game consists of 10 frames and each frame consists of 2 rolls
+* Bonus points are awarded for a strike or a spare
+* A strike occurs if all 10 pins are knocked down in roll 1 of the frame
+* A spare occurs if the total frame score is equal to 10
+
+##### WEB INTERFACE
+* Create a nice interactive animated interface with jQuery.
+
+### GETTING STARTED
+------------
+* JavaScript
+* Open folder in which you want to download project to
+
+```
+git clone git@github.com:rachjgriff/bowling-challenge.git
+```
+
+### TESTING
+------------
+* jasmine 3.4.0
+* To run from command line:
+```
+cd bowling-challenge/jasmine
+open SpecRunner.html
+```
+
+### USAGE
+------------
+
+* Google Chrome - right click - inspect
+* Navigate to console
+* Follow the below input in order to see how game works and the cumulative score is calculated:
+
+```
+var scorecard = new BowlingScorecard
+scorecard.rollScore(1);
+scorecard.rollScore(4);
+scorecard.game();
+scorecard.cumScore();
+```
+* If you want to see which frame you are on:
+```
+scorecard.frameCount;
+```
+
+* If you roll a spare or a strike in frame 10, follow the the below input in order to complete the game:
+
+```
+scorecard.rollScore(2);
+scorecard.rollScore(8);
+scorecard.game();
+scorecard.lastFrameSpareBonus(6);
+scorecard.cumScore();
+```
+```
+scorecard.rollScore(10);
+scorecard.game();
+scorecard.lastFrameStrikeBonus(6, 3);
+scorecard.cumScore();
+```
+
+### WEBSITE SPECIFICATION (REQUIREMENTS)
+------------
+- [x] User enters roll score
+- [x] Add roll score to frame score
+- [x] Add frame score to total score
+- [ ] Allow only 2 rolls to be added to a frame score
+- [ ] Allow only numbers 1 to 10 to be added as a roll score
+- [ ] Allow only 10 frame scores to be added to the total score
+- [ ] Count number of rolls
+- [x] Count number of frames
+- [x] If roll score 1 = 10, add it to the frame score and automatically set roll score 2 = 0
+- [x] If roll score 1 in previous frame = 10 and current frame roll 1 < 10, add current frame score twice to total score (Strike Bonus)
+- [x] if roll score 1 in previous frame = 10 and current frame roll 1 = 10, add current frame roll 1 and next frame roll 1 to total score (Strike Bonus)
+- [x] if roll score 1 in previous frame < 10 and previous frame score = 10, add roll score 1 of current frame twice to total score (Spare Bonus)
+- [x] If frame count = 10 and roll 1 score of frame 10 = 10, allow 2 more rolls and add these to frame 10 score
+- [x] if frame count = 10 and roll score 1 < 10 and frame 10 score = 10, allow 1 more roll and add this to frame 10 score
+
+### USER STORIES
```

Comment:
Great way to write user stories. Very detailed - truly guiding through the development process.